components/wifi/esp8266-driver/ESP8266/ESP8266.h

```diff
@@ -64,6 +65,15 @@
 #define FW_AT_LEAST_VERSION(MAJOR,MINOR,PATCH,NUSED/*Not used*/,REF) \
     (((MAJOR)*1000000+(MINOR)*10000+(PATCH)*100) >= REF ? true : false)
 
+struct esp8266_socket {
+    int id;
+    nsapi_protocol_t proto;
+    bool connected;
+    bool binded;
```

Comment:
Should it rather be "bound"? See [here for example](https://github.com/ARMmbed/mbed-os/blob/f469f710bca16abd1c538e48a3bee9db0a8e47e8/features/netsocket/NetworkStack.h#L281).

Reply:
Done